docs/overview.rst

```diff
@@ -89,10 +90,20 @@ You will get a runtime exception if you try to use it with an older cairo.
 You can however still use the rest of the API.
 There is no need for cairocffi’s versions to be tied to cairo’s versions.
 
-Use :func:`cairo_version` to test the version number::
+Use :func:`cairo_version` to test the version number for cairo::
 
     if cairo.cairo_version() > 11000:
         surface.set_mime_data('image/jpeg', jpeg_bytes)
+        
+Here are all the version numbers::
+
+      >>> c = str(cairo.cairo_version())
+      >>> print "The cairo version is", cairo.cairo_version(), " meaning ", c[0] + "." + c[1:3] + "." + c[3:]
+      The cairo version is 11402  meaning  1.14.02
+      >>> print "The pycairo verson is", cairo.version
+      The pycairo verson is 1.10.0
```

Comment:
Please explain a bit more here: “The latest pycairo version this cairocffi version is compatible with is”